kuma/users/models.py

```diff
@@ -195,6 +197,9 @@ def create_user_profile(sender, instance, created, **kwargs):
 
 @receiver(user_signed_up)
 def on_user_signed_up(sender, request, user, **kwargs):
+    context={'request':request}
+    msg='You have completed the first step of <a href="%s">getting started with MDN</a>' % wiki_url(context, 'MDN/Getting_started')
```

Comment:
To make this change, you could edit the file and add another commit. Once all of the changes have been made, we could squash all of the commits into one.